airflow/www/static/css/_page_dags.scss

```diff
@@ -0,0 +1,46 @@
+/**
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+.edit-dag {
```

Comment:
Is it worth scoping these  a bit more tightly if they are meant to just be for the dags page? i.e. changing the template to `<body class="dags">` (or on some other wrapper element) and then `.dags .edit-dag` etc?

Reply:
Other files do not have a specified scope.  I would rather not add new classes. The CSS code is much harder to predict then. I will add a `page_dags--` prefix to all classes from this files. What do you think about it?

Reply:
I'd rather `.page-dags .edit_dag` than `.page_dags--edit-dag`

Reply:
Can you tell me why your idea is better? My idea reduce specificy, so makes the code easier to understand.

Reply:
I don't have a hard-and-fast rule or way of saying it, but `.page_dags--edit-dag` feels like two it is actually two classes in one.

Given this is scss we could also do:

```scss
.page_dags {
     .edit_dag  {
        // ...
     }
}
```

Additionally having a `page_dags` or similar class at the top level helps with future plans I've got to allow custom/site-specific CSS theme (from a plugin in AIRFLOW_HOME) to build on top of the airflow base CSS.

Reply:
This is the BEM-like convention
Look at: 
https://css-tricks.com/bem-101/
https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/

My solution is compatible with the latest CSS code writing recommendations.  The articles are arguments for the introduction of my solution.

If we do not find any objective reasons, I will introduce my idea.

Reply:
It is worth adding that we have a very low quality CSS/JS code  Current changes should introduce the best trends for future changes.

Reply:
100% no to BEM.

`.btn__price`: Button ("btn") Does Not Belong In The Class.  That's like having `class="red"`. Css classes names should be semantic in meaning, not presentational.

There are hundreds of ways of writing CSS and an article saying that each of them is the best and everything else is wrong.

Specificity is an integral part of CSS. Let's use it, instead of working around it.

Going  back to the original comment: do these rules _need_ to be scoped to a single page, or can they just live in the main CSS bundle (in which case lets not have them in a file that implies they are specific to a single page)

Reply:
Thanks for discussion. I will update a PR with `..page_dags .edit_dag ` solution.

Reply:
I updated a PR.